src/containers/TriggerBindings/TriggerBindings.scss

```diff
@@ -0,0 +1,18 @@
+/*
+Copyright 2019 The Tekton Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+.triggerBindings {
```

Comment:
This is the same as the style for the trigger templates header. Can we remove this file and refactor the existing one so it can be reused more generically to ensure consistency across the pages?

Reply:
The reason behind respective css files is if in the future each page might have a unique style. If not I can go ahead and do the refactoring.

Reply:
If there are specific styles for each page that's fine to have the custom styles in a page-specific file, but for now they're identical so we should use common class names to ensure consistency

Overall we want all pages to feel like they're part of the same app (they are after all) so the more we can do to minimise all the subtle differences in spacing, size, layout, colour, etc. the better.